prov/rxm/src/rxm_conn.c

```diff
@@ -82,7 +82,8 @@ static int rxm_msg_ep_open(struct rxm_ep *rxm_ep, struct fi_info *msg_info,
 	}
 
 	if (!rxm_ep->srx_ctx) {
-		ret = rxm_ep_prepost_buf(rxm_ep, msg_ep);
+		dlist_init(&rxm_conn->posted_rx_list);
```

Comment:
Please move list initialization to rxm_conn creation, to avoid potential issues down the road.